packages/ERTP/src/paymentLedger.js

```diff
@@ -1,6 +1,5 @@
 // @ts-check
 
-import { assert, details as X } from '@agoric/assert';
```

Comment:
Using `assert` from global instead of import seems like a regression to me. If this is the recommended style, where is it documented?

Reply:
I agree it needs to be documented. Who wants to take this?

I know you didn't suggest it, but to be explicit, I do not think we should require this PR to provide that documentation.

Reply:
With those answered, I'm resolving this conversation. As always, if there's more to discuss, please unresolve it.